index.js

```diff
@@ -19,6 +19,38 @@ const manageRecipes = async () => {
     await Recipe.deleteMany();
 
     // Run your code here, after you have insured that the connection was made
+    
+    // Iteration 2 Create Recipe
+    let cozido = {
+      title: 'Cozido', 
+      level: 'Easy Peasy',  
+      ingredients: ['carnes', 'enchidos', 'feijão', 'arroz'], 
+      cuisine: 'Portuguese', 
+      dishType: 'main_course',  
+      image:'https://www.pingodoce.pt/wp-content/uploads/2015/03/cozido-a-portuguesa-617.jpg', 
+      duration: 180,
+      creator: 'Povo Português'
+    };
+     const recipe = await Recipe.create(cozido);
+     console.log(recipe);
+
+    //Iteration 3 - Insert multiple recipes
+    let allRecipes = await Recipe.insertMany(data);
+    console.log(allRecipes);
```

Comment:
If you wanted to `console.log` only the titles of the recipes, you needed to loop over your `allRecipes` variable that is storing an array with your recipes objects.
Something like this:
```
// console log to see the titles of the inserted recipes
    for (let i = 0; i < allRecipes.length; i++) {
      console.log(allRecipes[i].title);
    }
```